themes/sizes.js

```diff
@@ -74,4 +76,7 @@ export default {
   CONTAIN: 'contain',
   COVER: 'cover',
 
+  // OPACITY
+  OPACITY_08: 0.8,
```

Comment:
może opacities wywalimy z sizes - jakoś mi to nie gra ze sobą, np jakieś opacities.js albo coś